knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -95,6 +96,45 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 			return err
 		}
 	}
+
+	// Watch for kn ConsoleCLIDownload resources
+	knToRequestsFunc := handler.ToRequestsFunc(func(obj handler.MapObject) []reconcile.Request {
+		annotations := obj.Meta.GetAnnotations()
+		ownerNamespace := annotations[consoleclidownload.OwnerNamespace]
+		ownerName := annotations[consoleclidownload.OwnerName]
+		if ownerNamespace != "" && ownerName != "" {
+			return []reconcile.Request{{
+				NamespacedName: types.NamespacedName{Namespace: ownerNamespace, Name: ownerName},
+			}}
+		}
+		return nil
+	})
+
+	knManifest, err := consoleclidownload.RawManifest(mgr.GetClient())
+	if err != nil {
+		return err
+	}
+	knResources := knManifest.Resources()
+
+	gvkToCCD := make(map[schema.GroupVersionKind]runtime.Object)
+	for i := range knResources {
```

Comment:
Let's add the CCD itself to this list too. Changes to that should be reconciled by us as well, right?

Reply:
Yup, I'd added a separate watch for CCD, appended CCD in this list now.